dlk/python/dlk/core/operators.py

```diff
@@ -1446,7 +1446,7 @@ def run(self, **kwargs) -> Dict:
 
         n = 2 ** bit - 1
         np.clip(in_data, 0, max_value, out=in_data)
-        kwargs['data'] = np.round(in_data * n / max_value).astype(np.int32)
+        kwargs['data'] = np.floor(in_data * n / max_value + 0.5).astype(np.int32)
```

Comment:
@yd8534976 
I am not sure, I thought the problem is the `tf.round`, why you need to fix this, as it is using `np.round`.

Reply:
@nlpng Both `np.round` and `tf.round` round half to even, so I change both of them. And in `pass_lookup()`, this `run() func` will be called. Although in `pass_lookup()` it may have no obvious effect. We need to change it for consistency.